packages/twenty-front/src/modules/ai/components/internal/AgentChatContextRecordPreview.tsx

```diff
@@ -0,0 +1,47 @@
+import { RecordChip } from '@/object-record/components/RecordChip';
+import { AgentChatMultipleRecordPreview } from '@/ai/components/internal/AgentChatMultipleRecordPreview';
+import { useFindManyRecordsSelectedInContextStore } from '@/context-store/hooks/useFindManyRecordsSelectedInContextStore';
+import { useRecoilComponentValueV2 } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentValueV2';
+import { contextStoreCurrentObjectMetadataItemIdComponentState } from '@/context-store/states/contextStoreCurrentObjectMetadataItemIdComponentState';
+import { useObjectMetadataItemById } from '@/object-metadata/hooks/useObjectMetadataItemById';
+
+export const AgentChatContextRecordPreview = ({
+  agentId,
+}: {
+  agentId: string;
+}) => {
+  const { records, totalCount } = useFindManyRecordsSelectedInContextStore({
+    limit: 3,
+  });
+
+  const contextStoreCurrentObjectMetadataItemId = useRecoilComponentValueV2(
+    contextStoreCurrentObjectMetadataItemIdComponentState,
+  );
+
+  const { objectMetadataItem } = useObjectMetadataItemById({
+    objectId: contextStoreCurrentObjectMetadataItemId ?? '',
+  });
+
+  return (
+    <>
+      {/*{records.length < 3 ? (*/}
```

Comment:
I think we can delete this commented code as we have the context restore pattern it will likely never be multiple chips